bitrise.yml

```diff
@@ -529,16 +529,16 @@ workflows:
             App|${BITRISE_APP_URL}
             #mobile-testeng|https://mozilla.slack.com/archives/C02KDDS9QM9
             Mana|https://mana.mozilla.org/wiki/display/MTE/Mobile+Test+Engineering
-  4_B_xcode_build_and_test_Fennec_Enterprise_XCUITests:
+  4_B_xcode_build_and_test_FullFunctionalTestPlan:
```

Comment:
Not renaming these workflows would be confusing, but how could we know moving forward that the old and new names denote the same workflow?

Reply:
We may need a document with all the changes so that in the future we remember what we did and how. In this case the title should be` 4_B_xcode_build_and_test_Fennec`, we are building that scheme, then we use it to run different test plans.